static/scripts/audit-report/audit.ts

```diff
@@ -52,17 +55,63 @@ const tgBtnInput = document.querySelector("#cb4") as HTMLInputElement;
 
 let isCache = true;
 
-let isComment = true;
-let commentPageNumber = 1;
-
-type ListForRepoDataItem = Awaited<ReturnType<typeof octokit.rest.issues.listForRepo>>["data"][0];
-type ListCommentsDataItem = Awaited<ReturnType<typeof octokit.rest.issues.listComments>>["data"][0];
+// TODO: should be generated directly from the Supabase db schema
```

Comment:
I thought we already have something like a dynamically generated type file. Perhaps that's only in the bot source code?

Reply:
It is doable to call the Supabase to generate the types. But for this, you need a private key to access it, the public is not enough, afaik
https://supabase.com/docs/guides/api/rest/generating-types#generating-types-using-supabase-cli